frontend/packages/console-app/src/components/guided-tours/TourItemFooter.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Flex, FlexItem, Button } from '@patternfly/react-core';
+import { GuidedTourStatus } from './utils/guided-tour-status';
+
+type TourItemDescriptionProps = {
+  status: string;
+};
+const TourItemDescription: React.FC<TourItemDescriptionProps> = ({ status }) => (
+  <div className="odc-guided-tour__footer">
+    <Flex breakpointMods={[{ modifier: 'justify-content-space-between' }] as any}>
```

Comment:
The PF modifiers are a bit redundant but we should not cast to `any`:
```suggestion
    <Flex breakpointMods={[{ modifiers: FlexModifiers['justify-content-space-between']}>
```

Reply:
changed